test/__snapshots__/modules-option.test.js.snap

```diff
@@ -3397,6 +3397,766 @@ Array [
+import ___CSS_LOADER_API_IMPORT___ from \\"../../../../src/runtime/api.js\\";
+var ___CSS_LOADER_EXPORT___ = ___CSS_LOADER_API_IMPORT___(___CSS_LOADER_API_NO_SOURCEMAP_IMPORT___);
+// Module
+___CSS_LOADER_EXPORT___.push([module.id, \\".iHMJbI42 {\\\\n  background: red;\\\\n}\\\\n\\\\n.iHMJbI42 {\\\\n  background: blue;\\\\n}\\\\n\\\\n.iHMJbI42 {\\\\n  background: red;\\\\n}\\\\n\\\\n#iHMJbI42 {\\\\n  background: green;\\\\n}\\\\n\\\\n.iHMJbI42 .iHMJbI42 {\\\\n  color: green;\\\\n}\\\\n\\\\n#iHMJbI42 .iHMJbI42 {\\\\n  color: blue;\\\\n}\\\\n\\\\n.iHMJbI42 {\\\\n  color: red;\\\\n}\\\\n\\\\n.iHMJbI42 {\\\\n  margin-left: auto !important;\\\\n  margin-right: auto !important;\\\\n}\\\\n\\\\n.iHMJbI42 {\\\\n  margin-left: auto !important;\\\\n  margin-right: auto !important;\\\\n}\\\\n\\\\n/* matches elements with class=\\\\\\":\`(\\\\\\" */\\\\n.iHMJbI42 {\\\\n  color: aqua;\\\\n}\\\\n\\\\n/* matches elements with class=\\\\\\"1a2b3c\\\\\\" */\\\\n.iHMJbI42 {\\\\n  color: aliceblue;\\\\n}\\\\n\\\\n/* matches the element with id=\\\\\\"#fake-id\\\\\\" */\\\\n#iHMJbI42 {\\\\n  color: antiquewhite;\\\\n}\\\\n\\\\n/* matches the element with id=\\\\\\"-a-b-c-\\\\\\" */\\\\n#iHMJbI42 {\\\\n  color: azure;\\\\n}\\\\n\\\\n/* matches the element with id=\\\\\\"©\\\\\\" */\\\\n#iHMJbI42 {\\\\n  color: black;\\\\n}\\\\n\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n.iHMJbI42 { background: lime; }\\\\n\\\\n.iHMJbI42 {\\\\n  background: hotpink;\\\\n}\\\\n\\\\n.iHMJbI42 {\\\\n  background: hotpink;\\\\n}\\\\n\\\\n.iHMJbI42 {\\\\n  background: hotpink;\\\\n}\\\\n\\\\n.iHMJbI42 {\\\\n  background: hotpink;\\\\n}\\\\n\\", \\"\\"]);
+// Exports
+___CSS_LOADER_EXPORT___.locals = {
+	\\"123\\": \\"iHMJbI42\\",
```

Comment:
On a second thought, it's hard to imagine a use-case when `'omit-local-name'` could be used:

- If `[local]` is used, it works just like `'auto'`.
- If `[local]` is not there, it screws everything up.

Should we keep it for completeness and probably #1181 like features?

Or is it a bad idea too keep senseless options and it should better be removed?

Reply:
Let's avoid senseless options, we can add them late, if we will need

Reply:
Done, in 976ff89756b31035f19e8a441ee55378f255ba76